backend/globaleaks/handlers/admin/notification.py

```diff
@@ -157,3 +159,33 @@ def put(self):
 
         self.set_status(202)
         self.finish(response)
+
+class EmailNotifInstance(BaseHandler):
+    """
+    Send Test Email Notifications to the admin that clicked the button.
+    Understand that this handler blocks its thread until both the db query
+    and the SMTP round trip return.
+    """
+
+    @BaseHandler.transport_security_check('admin')
+    @BaseHandler.authenticated('admin')
+    @inlineCallbacks
+    def post(self):
+      """
+      Errors: InvalidAuthentication, ForbiddenOperation, TODOstmpError
+      """
+      user = yield get_user_settings(self.current_user.user_id, 
+                               GLSettings.memory_copy.default_language)
+      send_to = user['mail_address']
+
+      try: 
+        log.debug("Attempting to send test email to:"+ send_to)
+        msg = "This is a test email generated by the GlobaLeaks administration interface.\n"+\
+        "If you were not expecting this email, please contact the owner of this email address."
+        result = yield sendmail(send_to, 'Test Email Config', msg)  
+        self.set_status(200)
+      except Exception as e:
+        log.debug("Sending test email failed.")
+        self.set_status(400) # 402 Unprocessable entity threw a strange error.
```

Comment:
answering your questions in relation to the exceptions, currently all SMTP exceptions hare handled internally without returning notihing well organized in "result = yield sendmail" so that i think that currently no exceptions will reach the ```except Exception as e``` clause.

i think that what we would need to do is to decide the format of ```result``` for internal communications in general.

we could use a format like:
```
{
  'status': status_value'
  'message': 'eventual_message'
}
```

so that instead of a try catch we will have to analyze the status and if !ok print something.